msu/hooks/entity/tactical/player.nut

```diff
@@ -1,4 +1,13 @@
 ::mods_hookExactClass("entity/tactical/player", function(o) {
+	o.m.LevelUpsSpent <- 0;
+
+	local setAttributeLevelUpValues = o.setAttributeLevelUpValues;
+	o.setAttributeLevelUpValues = function( _v )
```

Comment:
Is this the vanilla parameter name again?

Reply:
Yes